crates/ruff/tests/lint.rs

```diff
@@ -45,9 +45,12 @@ inline-quotes = "single"
         success: false
         exit_code: 1
         ----- stdout -----
-        test.py:1:5: Q000 [*] Double quotes found but single quotes preferred
-        test.py:1:5: B005 Using `.strip()` with multi-character strings is misleading
-        test.py:1:19: Q000 [*] Double quotes found but single quotes preferred
+        error[Q000]: test.py:1:5: [*] Double quotes found but single quotes preferred
```

Comment:
It's not clear to me where and how we want to show the file name, location, and fixability